Services/DataX.Config/DataX.Config/ConfigGeneration/Processor/S500_GenerateProjectionFile.cs

```diff
@@ -23,12 +23,14 @@ public class GenerateProjectionFile: ProcessorBase
         public const string TokenName_ProjectionFiles = "processProjections";
 
         [ImportingConstructor]
-        public GenerateProjectionFile(IRuntimeConfigStorage runtimeStorage, IKeyVaultClient keyvaultClient)
+        public GenerateProjectionFile(IRuntimeConfigStorage runtimeStorage, IKeyVaultClient keyvaultClient, ConfigGenConfiguration conf)
```

Comment:
Can you add a test that validates the output of these steps for a given input of databricks options?

Reply:
Tests are already in place. We use the same config as used in hdinsight for databricks too.